src/components/molecules/ProfilePictureInput/ProfilePictureInput.tsx

```diff
@@ -152,7 +152,9 @@ const ProfilePictureInput: React.FunctionComponent<ProfilePictureInputProps> = (
       )}
       {isPictureUploading && <small>Picture uploading...</small>}
       {error && <small>Error uploading: {error}</small>}
-      <small>Or pick one from our Sparkle profile pics</small>
+      <small>
+        Feeling camera shy? Choose from one of these Summit profile avatars.
+      </small>
```

Comment:
[nit] styling
(the screenshot from the design)
![image](https://user-images.githubusercontent.com/63313732/122397117-a94d2e00-cf46-11eb-89cb-1a3183a1465d.png)

our version
![image](https://user-images.githubusercontent.com/63313732/122397970-70618900-cf47-11eb-9470-caf7affa5d50.png)


I don't think we need `<small >` here. Instead we need to increase margin and give `opacity: 0.8` to it.